pkg/handlers/cluster.go

```diff
@@ -0,0 +1,57 @@
+package handlers
+
+import (
+	"net/http"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	log "github.com/sirupsen/logrus"
+)
+
+type ClusterHandler struct {
+	ocm       *sdk.Connection
+	clusterId string
+}
+
+// For /api/clusters_mgmt/v1/clusters/{cluster_id}
+// https://api.openshift.com/#/default/get_api_clusters_mgmt_v1_clusters__cluster_id_
+func NewClusterHandler(o *sdk.Connection, clusterId string) *ClusterHandler {
+	log.Debug("Creating new cluster object Handler")
+	return &ClusterHandler{
+		ocm:       o,
+		clusterId: clusterId,
+	}
+}
+
+// https://pkg.go.dev/github.com/openshift-online/ocm-sdk-go@v0.1.382/clustersmgmt/v1#Cluster
+func (g *ClusterHandler) GetCluster(clusterID string) (*cmv1.Cluster, error) {
```

Comment:
Looks the `ClusterHandler` already contains a `clusterId` field. Is there a specific reason for having a different `cluster ID` to be passed in, the method could use the `clusterId` field directly?
```suggestion
func (g *ClusterHandler) GetCluster() (*cmv1.Cluster, error) {
	log.Debugf("Sending get cluster object request to OCM API: %s", g.clusterId)
	request := g.ocm.ClustersMgmt().V1().Clusters().Cluster(g.clusterId)
```

Reply:
That's true, I will remove the unnecessary parameter.